src/appleseed.python/bindutility.cpp

```diff
@@ -34,9 +34,59 @@
 // appleseed.foundation headers.
 #include "foundation/platform/python.h"
 
+// OIIO headers.
+#include "foundation/platform/_beginoiioheaders.h"
+#include "OpenImageIO/imagebufalgo.h"
+#include "foundation/platform/_endoiioheaders.h"
+
+// Standard headers.
+#include <sstream>
+
 namespace bpy = boost::python;
 using namespace foundation;
+using namespace OIIO;
 using namespace renderer;
+using namespace std;
+
+void oiio_make_texture(
+    const string&   in_filename,
+    const string&   out_filename,
+    const string&   in_colorspace,
+    const string&   out_depth)
+{
+    unordered_map<string, TypeDesc> out_depth_map{
+        {"sint8", TypeDesc::INT8},
+        {"uint8", TypeDesc::UINT8},
+        {"uint16", TypeDesc::UINT16},
+        {"sint16", TypeDesc::INT16},
+        {"half", TypeDesc::HALF},
+        {"float", TypeDesc::FLOAT}};
+
+    ImageSpec spec;
+
+    if (out_depth != "default")
+    {
+        spec.format = out_depth_map[out_depth];
+    }
+
+    spec.attribute("maketx:updatemode", 1);
+    spec.attribute("maketx:constant_color_detect", 1);
+    spec.attribute("maketx:monochrome detect", 1);
+    spec.attribute("maketx:opaque detect", 1);
+    spec.attribute("maketx:unpremult", 1);
+    spec.attribute("maketx:incolorspace", in_colorspace);
+    spec.attribute("maketx:outcolorspace", "linear");
+    spec.attribute("maketx:fixnan", "box3");
+
+    ImageBufAlgo::MakeTextureMode mode = ImageBufAlgo::MakeTxTexture;
+
+    stringstream s;
+
+    if (!make_texture(mode, in_filename, out_filename, spec, &s))
```

Comment:
I'm surprised that this works without the `ImageBufAlgo` namespace qualifier... How come?

Reply:
Not sure, ReSharper marked it as redundant and it compiled just fine without it.  Looking at the imagebufalgo header shows that it is in that namespace.

So I dunno.......

Reply:
What's even more surprising is that in other places such as `renderer/modeling/postprocessingstage/renderstamppostprocessingstage.cpp` where we use functions from `ImageBufAlgo`, removing the namespace qualifiers clearly no longer compiles...

Reply:
I can add it in just for consistency